weblab/fitting/views.py

```diff
@@ -197,3 +204,150 @@ def get(self, request, *args, **kwargs):
         }
 
         return JsonResponse(response)
+
+
+class FittingResultCreateView(LoginRequiredMixin, PermissionRequiredMixin, UserFormKwargsMixin, FormView):
+    """
+    Create and submit a fitting result from models, protocols, fitting specs and datasets
+    and (where relevant) their versions.
+    """
+    permission_required = 'fitting.run_fits'
+    form_class = FittingResultCreateForm
+
+    template_name = 'fitting/fittingresult_create_form.html'
+
+    def get_initial(self):
```

Comment:
Should we check visibility here, in case e.g. one user sends a URL to another that includes IDs the receiver isn't able to see?